test/groovy/UiVeri5ExecuteTestsTest.groovy

```diff
@@ -21,166 +21,45 @@ import util.JenkinsStepRule
 import util.Rules
 
 class UiVeri5ExecuteTestsTest extends BasePiperTest {
-    private ExpectedException thrown = ExpectedException.none()
+
```

Comment:
File can be removed completely, no tests required for the groovy part.

Reply:
File removed.